public/index.html

```diff
@@ -10,6 +10,11 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+
+      <!-- Import Google Icon Font -->
+      <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
same applies to these links

Reply:
@isheebo , I have implemented this feedback.